azure-pipelines.yml

```diff
@@ -26,6 +31,7 @@ stages:
             ARTIFACT_OS: macOS_azure
           linux_asan:
             imageName: 'ubuntu-20.04'
+            containerImage: $(MANYLINUX_IMAGE)
```

Comment:
Do we want to run the manylinux container for asan or just trying to have a generic linux test?

Reply:
I'll add a story to handle this. We are being forced to shuffle around our CI now because the backwards compatibility tests that run in the Ubuntu 20.04 Serialization test now take 1+ hours..

Reply:
Removed